packages/react-components/review/beta/react-components.api.md

```diff
@@ -637,6 +651,22 @@ export interface FileMetadata {
     url: string;
 }
 
+// @internal (undocumented)
+export const _FileUploadCards: (props: FileUploadCardsProps) => JSX.Element;
+
+// Warning: (ae-internal-missing-underscore) The name "FileUploadCardsProps" should be prefixed with an underscore because the declaration is marked as @internal
+//
+// @internal (undocumented)
+export interface FileUploadCardsProps {
```

Comment:
you have a linter warning here, id change the `FileUploadCardProps` name to `_FileUploadCardProps`

Reply:
done